schemas/post-artifact-response.yml

```diff
@@ -5,6 +5,52 @@ description: |
   Note that the `storageType` property is referenced in the request as well.
 type:               object
 oneOf:
+  - title: "Blob Artifact Response"
+    description: |
+      Response to a request for creating a new blob artifact
+    properties:
+      storageType:
+        description: |
+          Artifact storage type, in this case `'blob'`
+        type: string
+        enum:
+          - blob
+      expires:
+        description: |
+          Date-time after which the signed `requests` no longer work
+        type: string
+        format: date-time
+      requests:
```

Comment:
Couldn't we rename this `parts` ? and give each part a properties `offset` and `size`?